Dockerfile

```diff
@@ -44,15 +38,15 @@ RUN cd /opt/build && \
 #
 # Step 2 - build a lean runtime container
 #
-FROM alpine:${ALPINE_VERSION}
+FROM alpine:3.9
 
 ARG APP_NAME
 ENV APP_NAME=${APP_NAME}
 
 # Update kernel and install runtime dependencies
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
-    apk --no-cache add bash openssl
+    apk --no-cache add bash openssl erlang-crypto
```

Comment:
No `openssl-dev`? 😮

Reply:
My mistake!

Reply:
Wait! It’s on the build stage that `openssl-dev` is better thant `openssl`, not the runtime base image.